dbt/models/iasworld/schema/iasworld.dweldat.yml

```diff
@@ -1517,6 +1517,97 @@ sources:
                   config: *unique-conditions
                   meta:
                     description: yrblt should be between 1850 and now
+              - accepted_range:
+                  name: iasworld_dweldat_yrblt_205_more_than_62_years_old
+                  min_value: 1850
+                  max_value: cast(year(current_date) as decimal) - 62
```

Comment:
I used as decimal because that's how the >1850 accepted range is coded, but it doesn't seem like the most intuitive for dates.

Reply:
Hm, yeah -- I would go with an integer here if it produces the same behavior, even though the other test does it differently.